### DISCUSSION TITLE
Home assistant e Alexa

### DISCUSSION THREAD

**Discussion by ecaparco** [indomus/forum#637] [HUB: Home Assistant]
Salve a tutti, mi sto avvicinando al mondo degli hub personali e sono state molto utili tutte le varie guide e discussioni lette (conosco molto bene il "mondo" alexa, dispositivi per la domotica e quant altro), ma adesso vorrei approfondire il discorso degli hub personali, nello specifico home assistant. Per adesso su raspberry pi 4 ho installato come os HA e giocherellato con i vari dispositivi rilevati in casa. Il passaggio successivo è quello di "collegarlo" ad alexa. Ancora non ho avuto modo di provare, ma intanto avrei questo dubbio. In casa ho lampadine philips hue e osram (entrambi senza hub/gateway proprietario in quanto utilizzo un echo plus con protocollo zigbee). Ovviamente nessuna lampadina viene rilevata da HA. Nel momento in cui "collegherò" alexa ad HA mi ritroverò tutti i dispositivi che attualmente gestisco dall app alexa anche su HA? Attendo qualche vostra delucidazione in merito. Grazie!

Amedeo

**Comment by snakuzzo** [comment]
no.
Puoi esporre i device da HA verso Alexa, non il contrario.
Se vuoi passare agli hub personali come HA, comincia a dismettere l'hub zigbee di Alexa e ad usarne uno universale magari con la penna di @gio-dot che fa da coordinator e software come zigbee2mqtt o ZHA

**Comment by ecaparco** [comment]
Ciao, grazie per la risposta che "purtroppo" immaginavo. A questo punto però mi sorge un altro dubbio, anche se tecnicamente non credo sia possibile. Potrei lasciare le lampadine collegati ad entrambe gli hub in modo da sfruttarli da tutti e due i sistemi, alexa ed eventuale ZHA?

**Comment by snakuzzo** [comment]
Non puoi integrarle su hub diversi e cmq non ha senso. Una volta integrate su HA le esponi verso Alexa
